src/routes/feed.xml/+server.ts

```
@@ -98,12 +98,21 @@ ${events
 		const eventDate = new Date(event.start);
 		const locationParts = [event.city?.name, event.city?.country].filter(Boolean);
 		const location = locationParts.join(', ');
+		// eslint-disable-next-line no-restricted-syntax -- RSS server endpoint, no request locale; fixed en-US by design
+		const eventDateFormatted = event.start
+			? eventDate.toLocaleDateString('en-US', {
+					weekday: 'long',
+					year: 'numeric',
+					month: 'long',
+					day: 'numeric'
+				})
```

ESLint

[error] Do not format dates with toLocale* directly. Use a helper from $lib/utils/date.ts (or calendar.ts) so dates follow the UI language with textual months. See CLAUDE.md "Date & Time Formatting".

(no-restricted-syntax)